src/Infrastructure/OrchestratorService.cs

```diff
@@ -46,6 +46,17 @@ public async Task FetchMapAndSaveEmployeeData()
 
         var phoneNumberUtil = PhoneNumberUtil.GetInstance();
 
+        foreach (var bemanning in bemanningEntries.Where(employee => !IsActiveEmployee(employee)))

```

Comment:
This loop also takes over the loop at the end of the function which removes employees with an endDate so the other one can be deleted.

Reply:
That's a good point. Might actually just change that to have `EnsureInactiveEmployeesAreDeleted` 🤔 


The consequence though, is that employees before the start date won't be able to add contact information and allergies. That's not good either. But this is also true with existing implementation

Reply:
Refactored and fixed deleting blobs for both